src/simd/sq8_simd_test.cpp

```diff
@@ -54,11 +54,11 @@ TEST_CASE("SQ8 SIMD Compute Codes", "[SQ8 SIMD]") {
     std::vector<uint8_t> vec1, vec2;
     for (const auto& dim : dims) {
         auto vec = fixtures::generate_vectors(count * 2, dim);
-        vec1.resize(count * dim * 2, 0);
+        vec1.resize(count * dim);
```

Comment:
modify all self-defined dims to common_used_dims

Reply:
done